src/core/runtime.ts

```diff
@@ -160,6 +161,19 @@ class Runtime {
         }
     }
 
+    private handlePageScrollPosition(): void {
+        if (window.location.hash) {
```

Comment:
Will this take into account any statically set nav bars or other elements?

Reply:
We'll address this in #42